presto-orc/src/main/java/com/facebook/presto/orc/OrcWriterOptions.java

```diff
@@ -50,6 +50,7 @@
     private final OptionalInt compressionLevel;
     private final StreamLayout streamLayout;
     private final boolean integerDictionaryEncodingEnabled;
+    private final boolean stringDictionarySortingEnabled;
```

Comment:
This will have lot of merge conflict with https://github.com/prestodb/presto/pull/16089, but depending on which one gets in first, I will resolve the merge conflicts.